DocsCode/NavigationTutorial/NavigationTutorialViewController.swift

```diff
@@ -41,27 +41,18 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         let point = sender.location(in: mapView)
         let coordinate = mapView.convert(point, toCoordinateFrom: mapView)
 
-        // Create a basic point annotation and add it to the map
-        let annotation = MGLPointAnnotation()
-        annotation.coordinate = coordinate
-        annotation.title = "Start navigation"
-        mapView.addAnnotation(annotation)
-
-        // Calculate the route from the user's location to the set destination
-        calculateRoute(from: (mapView.userLocation!.coordinate), to: annotation.coordinate) { (route, error) in
-            if error != nil {
-                print("Error calculating route")
-            }
+        if let origin = mapView.userLocation?.coordinate {
```

Comment:
nit:
```suggestion
        if let currentUserLocation = mapView.userLocation?.coordinate {
```

and then we can remove the comment below which is a bit excessive.

Reply:
Same changes were already made in docs [here](https://github.com/mapbox/help/pull/2648). I'd rather keep this as it is to prevent adding nitpicking changes in `help` once more.